collections.sh

```diff
@@ -3,10 +3,7 @@ set -e
 
 docker_publish()
 {
-  echo "START: Publish.sh"
   sh ./scripts/publish.sh
-  echo "END: Publish.sh"
-  echo "START: trivy_scan.sh"
-  sh ./scripts/trivy_scan.sh
-  echo "END: trivy_scan.sh"
```

Comment:
Why are you removing trivy_scan?

Reply:
I was told by @naveen-variant that publish.sh already runs trivy_scan.sh. Is trivy_scan.sh used differently in both secnarios?